### PR TITLE
Respect required-resource-attributes

### LINKED ISSUE #47: Respect required-resource-attributes from l10n.yaml
When `required-resource-attributes` is `false` or does not exist, don't generate empty attributes in output ARB.

### PR DESCRIPTION
**Pull request by Albert221**
Resolves #47.

### CODE REVIEW

File: converter/testdata/attributes-req-attrs.input [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    {
+        "term": "attrs",
+        "definition": "Foo {bar}",
+        "context": "",
+        "term_plural": "",
+        "reference": "",
+        "comment": ""
+    }
+]
```
Comment (bartekpacia) [inline converter/testdata/attributes-req-attrs.input:18]: missing newline

File: converter/testdata/attributes.input [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    {
+        "term": "attrs",
+        "definition": "Foo {bar}",
+        "context": "",
+        "term_plural": "",
+        "reference": "",
+        "comment": ""
+    }
+]
```
Comment (bartekpacia) [inline converter/testdata/attributes.input:18]: \n

**Review by bartekpacia — APPROVED** [review]

File: converter/testdata/attributes-req-attrs.input (hunk 1 above)
Comment (Albert221) [reply converter/testdata/attributes-req-attrs.input:18]: POEditor doesn't include newline in those files afair

Comment (bartekpacia) [reply converter/testdata/attributes-req-attrs.input:18]: ah, right, sorry

**Pull request merged**